### DISCUSSION TITLE
Object type attribute null value

### DISCUSSION THREAD

**Discussion by goto3** [sensedeep/dynamodb-onetable#416] [Q&A] [answered]
Hello, if I set a model attribute as Object, I can not set it to null or undefined, the empty value is `{}` which is confusing once fetching from the database.

```js
models: {
    LoyaltyProgram: {
      pk: { type: String, value: 'lp:${id}' },
      sk: { type: String, value: 'lp:${businessUnitId}' },
      id: { type: String, generate: 'uuid' },      
      accumulationUnit: {
        type: Object,
        default: undefined,
        nulls: true,
        schema: {
          name: { type: String },
          code: { type: String },
          baseFactor: { type: Number },
        },
      },
    },
}
```

if I do a `model.create({... , accumulationUnit: null });` the return value once I fetch from the database for the accumulationUnit should be null or undefined but it returns an empty object `{}`
![image](https://user-images.githubusercontent.com/16456164/200853427-9efe2a11-0a06-4534-a7b4-7f8ea4b9e9ad.png)

**Comment by mobsense** [comment]
That could be an issue. Can you create a test case for us to use when debugging and fixing?

See the issue template. This saves us lots of time for folks to follow the issue template and create a test case for us.

**Comment by dev-embedthis** [comment, marked as answer]
I think this has been addressed in 2.6.x
